### DISCUSSION TITLE
Config file isn't being recognised

### DISCUSSION THREAD

**Discussion by hrai** [BurntSushi/ripgrep#2323] [Q&A] [answered]
I've set the env var for the config.

`env | rg ripgreprc` returns `RIPGREP_CONFIG_PATH=~/.ripgreprc`

`less ~/.ripgreprc` returns the file data.

`rg blah` returns `~/.ripgreprc: No such file or directory (os error 2)`

I can't figure out what's going on.

**Comment by BurntSushi** [comment, marked as answer]
`~` is expanded by your shell, but ripgrep isn't your shell and doesn't expand it. I would normally expect it to be expanded as part of setting your environment variable here, but for whatever reason, it isn't.

I would suggest typing out the full path. If that works, then you know your issue is with `~` not being expanded.

**Comment by hrai** [comment]
I had to use `$HOME/.ripgreprc` and it's fine now.
